x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/activity_monitor/index.tsx

```diff
@@ -0,0 +1,337 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiBasicTable, EuiPanel, EuiSpacer } from '@elastic/eui';
+import React, { useState } from 'react';
+import { HeaderSection } from '../../../../components/header_section';
+import {
+  UtilityBar,
+  UtilityBarAction,
+  UtilityBarGroup,
+  UtilityBarSection,
+  UtilityBarText,
+} from '../../../../components/detection_engine/utility_bar';
+import { columns } from './columns';
+
+export interface RuleTypes {
+  href: string;
+  name: string;
+}
+
+export interface ColumnTypes {
+  id: number;
+  rule: RuleTypes;
+  ran: string;
+  lookedBackTo: string;
+  status: string;
+  response: string | undefined;
+}
+
+export interface PageTypes {
+  index: number;
+  size: number;
+}
+
+export interface SortTypes {
+  field: string;
+  direction: string;
+}
+
+export const ActivityMonitor = React.memo(() => {
+  const sampleTableData = [
```

Comment:
for readability can this object live somewhere else

Reply:
This was a copy-pasta move of the initial `Activity Monitor` table placeholder to its own component. Happy to move it elsewhere till we have a chance to work it, but it sounds like this table might be removed altogether?

Reply:
ok so forget about it